src/main/java/frc/robot/RobotContainer.java

```diff
@@ -4,39 +4,33 @@
 
 package frc.robot;
 
+//import edu.wpi.first.wpilibj.PowerDistribution;
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj.GenericHID.RumbleType;
 import edu.wpi.first.wpilibj2.command.Command;
 
 import frc.robot.Constants.ControllerConstants;
-import frc.robot.commands.DrivebaseTeleop;
-import frc.robot.subsystems.Climber;
+import frc.robot.commands.curveDrive;
 import frc.robot.subsystems.Drivebase;
-import frc.robot.subsystems.Intake;
-import frc.robot.subsystems.Shooter;
-import frc.robot.subsystems.Vision;
+
 
 public class RobotContainer {
+
   private final Drivebase drivebase = new Drivebase();
-  private final Intake intake = new Intake();
-  private final Shooter shooter = new Shooter();
-  private final Climber climber = new Climber();
-  private final Vision vision = new Vision();
   
   private final XboxController controller = new XboxController(ControllerConstants.CONTROLLER_PORT);
 
-  public RobotContainer() {
-    drivebase.setDefaultCommand(new DrivebaseTeleop(
-      drivebase,
-      () -> applyDeadband(-controller.getLeftY()),
-      () -> applyDeadband(-controller.getRightX()),
-      () -> applyDeadband(controller.getRightTriggerAxis())
-    ));
+  //private final PowerDistribution PDP = new PowerDistribution();
 
+  public RobotContainer() {
+    drivebase.setDefaultCommand(new curveDrive(drivebase, () -> controller.getRawAxis(ControllerConstants.SPEED_JOYSTICK), () -> controller.getRawAxis(ControllerConstants.ROTATION_JOYSTICK), () -> controller.getRawButtonPressed(ControllerConstants.ROTATE_IN_PLACE_BUTTON), () -> controller.getRawAxis(ControllerConstants.THROTTLE_TRIGGER)));
```

Comment:
Change this to Xbox/PS4 controller - this can be done in future change however given that it works